packages/react-next/src/components/Dropdown/Dropdown.test.tsx

```diff
@@ -92,18 +93,21 @@ describe('Dropdown', () => {
     });
 
     it('Renders a selected item if option specifies selected', () => {
-      wrapper = mount(
+      safeCreate(
         <Dropdown
           label="testgroup"
           options={[
             { key: '1', text: '1', selected: true },
             { key: '2', text: '2' },
           ]}
         />,
+        container => {
+          const dropdownOptionText = container.root.find(node => {
```

Comment:
I know you mentioned this in the past but remind me why doing this with enzyme didn't work? It seems kind of unfortunate to switch the tests over to react-test-renderer since the syntax is so much more verbose and doing updates is more error-prone (since you have to manually ensure all the same props are passed except for the one you meant to change).

Reply:
In this case, I'm not sure why I changed it. I've reverted it since it works the old way.

The other cases below were using internal state, which isn't exposed on functional component. Additionally, I think hooks are not run (or run consistently) when using enzyme.

Reply:
You should be able to get hooks to work by wrapping certain operations in `act()` (from `react-dom/test-utils`), though if I remember right it can still be a bit challenging to get certain things working.